testplan/web_ui/testing/src/Nav/InteractiveTreeView.js

```diff
@@ -0,0 +1,326 @@
+import React from 'react';
```

Comment:
This is essentially duplication of the TreeView, but entries are interactive entries inside each node.